akka-actor-typed/src/main/scala/akka/actor/typed/Behavior.scala

```diff
@@ -222,6 +222,26 @@ object Behavior {
       case other                         ⇒ other
     }
 
+  /**
+   * Return special behaviors, undefer deferred, if behavior is non-special, apply the wrap function to get
+   * and return the result from that.
+   *
+   * INTERNAL API (for now)
+   */
+  // FIXME not an amazing name this
+  // FIXME should it be public?
```

Comment:
would that be needed if someone implement some custom behavior like `intercept`?
gut feeling is that it could wait until needed (requested)

Reply:
Good. I agree on deferring opening it up.